pyro/distributions/von_mises.py

```diff
@@ -79,3 +120,11 @@ def expand(self, batch_shape):
             loc = self.loc.expand(batch_shape)
             concentration = self.concentration.expand(batch_shape)
             return type(self)(loc, concentration, validate_args=validate_args)
+
+    @property
+    def mean(self):
```

Comment:
It's worth noting that this is the circular mean rather than euclidean mean, either in the main class docstring or here. I think you can add an `:attr:` field to the main docstring, as we've done in a couple other places.

Reply:
Thanks for the comments, I will look into how to do this.